### PR TITLE
Task-48085: Task assigning is not working.

### PR DESCRIPTION
**Pull request by lbengayed**
In the taskDrawer the receive event was updateTaskAssignment, so we should change the name of emitted event in taskAssignment to updateTaskAssignment.

### CODE REVIEW

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you explain your modification ?

You should fill the PR description and the commit message to answer WHY ? and HOW ? questions.

Thanks

File: task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue [hunk 1]
```diff
@@ -103,7 +103,7 @@
           <div class="taskAssignement ms-8 pb-3">
             <task-assignment
               :task="task"
-              @updateTaskAssignment="updateTaskAssignee($event)"
+              @updateTaskAssignement="updateTaskAssignee($event)"
```
Comment (azayati) [inline task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue]: In english it should be "updateTaskAssignment", so I think that you should change the name of emitted event instead

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you update your commit message and PR description to follow the commit message best practises :
https://developer.exoplatform.org/#id-commits-messages-best-practices
The commit message should be

Title
Empty line
Description answering question WHY ? and HOW ?

Thanks

**Review by rdenarie — DISMISSED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Review by azayati — CHANGES_REQUESTED** [review]
@lbengayed https://github.com/exoplatform/task/blob/8e3bf1f92cde73889972cd967ff0cab80fdee47d/task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskAssignment.vue#L275 you have also this reference to updateTaskAssignement, should it be changed also ?

**Comment by lbengayed** [comment]
> @lbengayed
> 
> https://github.com/exoplatform/task/blob/8e3bf1f92cde73889972cd967ff0cab80fdee47d/task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskAssignment.vue#L275
> 
> you have also this reference to updateTaskAssignement, should it be changed also ?

yes thnx @azayati

**Review by azayati — CHANGES_REQUESTED** [review]
@lbengayed 
When seeing commits I think that this commit https://github.com/exoplatform/task/commit/8e3bf1f92cde73889972cd967ff0cab80fdee47d#diff-d684f376b0426354ae571baaa57cad44ecf876ae10e59323455ca4762c3c3c0e has overriden some files of the commit just before, can you verify them please especially TasksViewDashboard.vue TaskPriority.vue? cc @SaraBoutej

**Comment by lbengayed** [comment]
> TasksViewDashboard.vue

@azayati the root cause of this bug was this commit [8e3bf1f#diff-d684f376b0426354ae571baaa57cad44ecf876ae10e59323455ca4762c3c3c0e](https://github.com/exoplatform/task/commit/8e3bf1f92cde73889972cd967ff0cab80fdee47d#diff-d684f376b0426354ae571baaa57cad44ecf876ae10e59323455ca4762c3c3c0e) , i fix the problems of priority and assigning

**Review by rdenarie — DISMISSED** [review]

**Review by azayati — DISMISSED** [review]
LGTM

@lbengayed we should as recommended verify the last commit with @SaraBoutej to avoid other regressions cc @rdenarie

**Review by SaraBoutej — DISMISSED** [review]
LGTM

**Review by SaraBoutej — APPROVED** [review]

**Review by azayati — CHANGES_REQUESTED** [review]
@lbengayed the PR should be created on feature/maintenance before

**Pull request closed without merging**